packages/flutter_tools/lib/src/windows/windows_version_validator.dart

```diff
@@ -0,0 +1,79 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:process/process.dart';
+
+import '../base/io.dart';
+import '../doctor_validator.dart';
+
+const List<String> unsupportedVersions = <String>[
+  '6',
+  '7',
+  '8',
+];
+
+/// Validator to be run with `flutter doctor` to check
+/// Windows host machines if they are running supported versions.
+class WindowsVersionValidator extends DoctorValidator {
```

Comment:
```suggestion
/// Validator for supported Windows host machine operating system version.
class WindowsVersionValidator extends DoctorValidator {
```

Since all validators are used by doctor